lib/Widget/notification/all_Notification_Table.dart

```diff
@@ -0,0 +1,556 @@
+                                    onPressed: () {
+                                      Navigator.pop(context);
+                                    },
+                                    child: Text('ยกเลิก',
+                                        style: TextStyle(fontSize: 16))),
+                              ),
+                              Container(
+                                width: 150,
+                                height: 40,
+                                margin: EdgeInsets.only(left: 20),
+                                padding: const EdgeInsets.only(left: 0),
+                                child: ElevatedButton(
+                                    style: ElevatedButton.styleFrom(
+                                        primary: Color(0xFF2ED47A),
+                                        onPrimary: Colors.white,
+                                        padding: EdgeInsets.all(15)),
+                                    onPressed: () {
+                                      _firebaseService
+                                          .updateDataToCollectionField(
+                                              collection: 'Notifications',
+                                              docId: notiId,
+                                              data: {'seen': true});
+                                      Navigator.pop(context);
+                                    },
+                                    child: Text('ดำเนินการแล้ว',
+                                        style: TextStyle(fontSize: 16))),
+                              ),
+                            ],
+                          );
+                        } else {
+                          return Row(
+                            mainAxisAlignment: MainAxisAlignment.center,
+                            children: [
+                              Icon(
+                                Icons.check_circle_outline_rounded,
+                                size: 40,
+                                color: Colors.greenAccent,
+                              ),
+                              Text(" $seen",
+                                  style: TextStyle(
+                                      color: Color(0xFF2ED47A), fontSize: 24),
+                                  textAlign: TextAlign.center),
+                            ],
+                          );
+                        }
+                      }()),
+                    )
+                  ],
+                ),
+              );
+            },
+          ),
+        );
+      },
+    ).then((value) => widget.callAllData(widget.patientState));
+  }
+
+  Future<void> alertSuccessfullyChangeStatusPostHome(
+      BuildContext context,
+      notiId,
+      formName,
+      hn,
+      formTime,
+      formDate,
+      name,
+      imgURL,
+      seen,
+      serverity,
+      advice) async {
+    final IFirebaseService _firebaseService = locator<IFirebaseService>();
```

Comment:
Shouldn't define it again in the method if in its class has already called it

Reply:
Solved  in 175e0ed kub